dev/NumberBox/NumberBox.idl

```diff
@@ -60,6 +60,7 @@ unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
     String PlaceholderText;
     Windows.UI.Xaml.Controls.Primitives.FlyoutBase SelectionFlyout;
     Windows.UI.Xaml.Media.SolidColorBrush SelectionHighlightColor;
+    Windows.UI.Xaml.TextAlignment TextAlignment;
```

Comment:
Posting previous comment here as this is more suitable in the IDL file.

I think this would technically go through API review first. @MikeHillberg Should this be marked as preview until then?

Reply:
Yes, this should be reviewed and marked as preview because of an [apparent mismatch](https://github.com/microsoft/microsoft-ui-xaml/issues/2896) between the documentation and the API behavior regarding the TextBox.HorizontalTextAlignment and TextBox.TextAlignment APIs.

Reply:
@sonnemaf until this has an approved spec, you'll need to mark the API as preview, examples can been seen in some other IDL files in the repo

Reply:
@sonnemaf For example [here](https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/Repeater/ItemsRepeater.idl#L439).

Reply:
Done!